app/views/conversations/show.html.erb

```diff
@@ -1,7 +1,14 @@
 <%= turbo_stream_from @conversation %>
-<div class="flex h-screen" data-controller="click-toggle" data-click-toggle-flippable-class="md:flex">
+<div class="flex h-screen"
+     data-controller="click-toggle"
+     data-click-toggle-flippable-class="md:flex"
+     data-action="keydown.meta+shift+s@document->click-toggle#handleEvent:prevent"
```

Comment:
I just did a little research to see if there is a more standard one:

* Slack uses `Cmd + .` for the right sidebar and `Cmd Shift D` for the left nav bar
* Notion uses `Cmd + .` for the left nav bar -or- `Cmd \` (they seem to more prominently advertise the `\` even though I've always used period)
* Visual Studio uses `Cmb + B` to open and close the sidebar

...I really wish there was a more standard shortcut like there is for focusing within the search bar.